cmd/podman/trust_set_show.go

```diff
@@ -254,15 +254,16 @@ func getPolicyJSON(policyContentStruct trust.PolicyContent, systemRegistriesDirP
 			policyJSON[repo]["type"] = repoval[0].Type
 			policyJSON[repo]["transport"] = transname
 			keyarr := []string{}
-			uids := []string{}
+			// TODO - uids is never used.  i have no idea about the intent; commenting out to fix later
```

Comment:
@QiWang19 PTAL at these.  I believe they can just be removed.

Reply:
Yes. This can be removed.